Code snippets/src/main/java/data_access/DBNotesDataAccessObject.java

```diff
@@ -309,6 +310,7 @@ public boolean existsByName(String identifier) {
     }
```

Comment:
I see that the location and indentation of some curly braces can be changed to meet the better coding style.